integration/tests/posit/connect/test_content.py

```diff
@@ -23,7 +23,7 @@ def test_count(self):
         assert self.client.content.count() == 1
 
     def test_get(self):
-        assert self.client.content.get(self.content.guid) == self.content
```

Comment:
I'm reaching the edge of my python knowledge here so maybe this is a dumb question but is `.guid` now a deprecated syntax and did that happen in this PR?

Reply:
It's self-imposed. See https://github.com/posit-dev/posit-sdk-py/issues/262

We are moving away from attribute-based access for object data.